antlr/PlantUML.g4

```diff
@@ -133,11 +133,13 @@ ignore:
     |
     'scale' HWS rest_of_line
     |
+    'title' HWS rest_of_line
```

Comment:
This looks good! 🎉 Is it possible to omit the other white space changes in this file though? Ideally the PR would only include changes for the `title` feature. Which editor and settings are you using?

Reply:
using VSCode, enabled setting to "Trim Trailing WhiteSpace". 
I would suggest that setting. 

Will there ticket to remove those trailing white spaces?

Reply:
I added https://github.com/StateSmith/StateSmith/issues/218 to figure this out and discuss. It is a good point. Once we settle on a direction we should make that change to all files in a dedicated PR.